pages/portfolio.html

```diff
@@ -1,12 +1,35 @@
+<!-- must include information about several projects you've completed at Ada (or elsewhere) with links to the GitHub repo if available, descriptions, images, etc. -->
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Alma's portfolio</title>
+  <link href="styles/style.css" rel="stylesheet">
 </head>
-<body>
-  
+<body class="container">
+  <section>
```

Comment:
Because this `section` is the only element inside your container grid, the grid doesn't get applied correctly. Direct children of the grid container are the ones that get ordered according to grid rules.